### PR TITLE
New blog contribution guide

### PR DESCRIPTION
**Pull request by bennyvasquez**
This is just an update to the CONTRIBUTING.md to add how to contribute blog posts. I thought I'd submitted this already, but apparently not!

### CODE REVIEW

**Comment by codyro** [comment]
Can I add some formatting to your branch? The content would remain the same, but things like filenames I'll tidy up (EX, blah/partial/single.html -> `/blah/partial/single.html`).

Regarding the content, I think we should clarify a few things:

> by submitting content to the AlmaLinux website, you are affirming that you are the sole author of all text and images used in the post, and that you own all rights to publish said content.

Should we require any content/assets under specific licenses?

> external links are allowed, but links to paywalled websites or obvious use of a blogpost on the AlmaLinux site as an SEO builder for another website is prohibited.

We need to be *very* careful here. We should specify that all external links will be `nofollow`ed (https://en.wikipedia.org/wiki/Nofollow) and will need to be reviewed. This should curb any SEO/spam content from the get-go. 

Also, a space between `blog` & `post` :).

My other suggestion would be to link to the https://github.com/AlmaLinux/almalinux.org/tree/master/content/blog page for examples of live blog posts.

**Comment by bennyvasquez** [comment]
Great feedback, @codyro. Mind another look, when you get a second?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,38 +2,94 @@
...
+
+- Below the --- of the metadata block, you should place the content of your blog post. This content should be in markdown format, and any images you want to include in your post can be added to `./static/blog-images/` and will serve from `/blog-images/`.
+- Before submitting your PR, please test that your content loads correctly by building the site locally with the command `hugo server`
+
+
+### Some notes about content
+
+- define all terms used (especially acronyms) so that all readers may understand your content. 
+- content submitted to the AlmaLinux blog should be unique and not posted elsewhere on the internet.
+- by submitting content to the AlmaLinux website, you acknowledge that you are aware of our [https://almalinux.org/p/the-almalinux-os-licensing-policy/license](license policy), are affirming that you own all rights to publish said content.
```
Comment (codyro) [inline CONTRIBUTING.md:64]: Broken URL/markdown:

> - by submitting content to the AlmaLinux website, you acknowledge that you are aware of our [https://almalinux.org/p/the-almalinux-os-licensing-policy/license](license policy), are affirming that you own all rights to publish said content.

to

> - by submitting content to the AlmaLinux website, you acknowledge that you are aware of our [license policy](https://almalinux.org/p/the-almalinux-os-licensing-policy/license), are affirming that you own all rights to publish said content.

**Review by codyro — CHANGES_REQUESTED** [review]
Couple of missed style tweaks/broken hyperlink/wrong `hugo serve` command. Looks good otherwise!

File: CONTRIBUTING.md (hunk 1 above)
Comment (codyro) [reply CONTRIBUTING.md:64]: Oops, looks like that URL is a 404 as well, so that needs be adjusted too.

**Comment by bennyvasquez** [comment]
@codyro all resolved!

**Review by codyro — APPROVED** [review]

**Comment by codyro** [comment]
Looks good to merge to me!

**Pull request merged**